app/assets/stylesheets/application.css.scss

```diff
@@ -0,0 +1,159 @@
+$main-font: 'Josefin Sans', sans-serif;
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.layouts {
+  background-image: url("blue-background.jpg");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+
+  h1 {
+    color: #fff;
+  }
+
+  h2 {
+    color: #fff;
+  }
+}
+
+.arrow {
+  color: #fff;
+  margin: 2%;
+  text-decoration: none;
+}
+
+#error_explanation {
+  color: #fff;
+  display: block;
+  background-color: rgba(255, 0, 0, 0.6);
+  padding: 2%;
+  border-radius: 10px;
+  overflow: hidden;
+
+  ul {
+    list-style: none;
+    margin: 0 0 18px 0;
+  }
+}
+
+.parent {
+  height: 100vh;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  font-family: $main-font;
+
+  form {
+    margin: 0 auto;
+    display: flex;
+    justify-content: center;
+
+    label {
+      color: #fff;
+    }
+
+    input {
+      border-radius: 10px;
+      border-style: none;
+    }
+
+    h2 {
+      color: #fff;
+    }
+
+    .actions {
+      margin: 0 auto;
+    }
+  }
+
+  .card {
+    width: 300px;
+    height: 300px;
+    color: #000;
+    background-color: #3979c3;
+    border-radius: 10px;
```

Comment:
**Notice**: here are the colors to be used for your design
![Screenshot from 2022-10-21 18-13-13](https://user-images.githubusercontent.com/83042742/197241954-2cfcdc72-c029-4ec3-88ef-e2b4efad91c0.png)